docs/pages/billing/manage.mdx

```diff
@@ -0,0 +1,262 @@
+
+<ImageSpotlight
+  alt="On Cart, select On-demand plan under Choose a plan and Continue to Review at the bottom of the page in Expo dashboard."
+  src="/static/images/billing/management/billing-06.png"
+  style={{ maxWidth: 640 }}
+/>
+
+- Next, you need to review and confirm your new plan. Verify that the account you're downgrading from is correct (see the _warning_ on this page). After reviewing the details, enable the checkbox and click **Continue to Checkout**.
+
+<ImageSpotlight
+  alt="Billing page in Expo dashboard displays the review page for the selected plan to downgrade."
+  src="/static/images/billing/management/billing-07.png"
+  style={{ maxWidth: 640 }}
+/>
+
+- A **Checkout** popup opens up where you can confirm all the details. Click **Schedule Downgrade** to schedule a downgrade of your plan.
+
+<ImageSpotlight
+  alt="Billing page in Expo dashboard displays the review page for the selected plan to downgrade."
+  src="/static/images/billing/management/billing-08.png"
+  style={{ maxWidth: 280 }}
+/>
+
+- After confirming your account for a plan downgrade to On-demand, the same information is also reflected under **Billing** > **Plan**.
+
+<ImageSpotlight
+  alt="Billing page in Expo dashboard displays the review page for the selected plan to downgrade."
+  src="/static/images/billing/management/billing-09.png"
+  style={{ maxWidth: 640 }}
+/>
+
+</Tab>
+
+<Tab label="From Production, Enterprise or Legacy to Free plan">
+
+Downgrading from a Production or an Enterprise plan to the Free plan takes effect after your current billing cycle ends.
+
+To downgrade, go to [**Billing**](https://expo.dev/settings/billing) and follow the steps below:
+
+- Under **Plan**, click **Change Plan**.
+
+<ImageSpotlight
+  alt="Billing page in Expo dashboard displays the Change Plan button to downgrade a plan."
+  src="/static/images/billing/management/billing-05.png"
+  style={{ maxWidth: 720 }}
+/>
+
+- Under **Choose a plan**, select **Free plan**. Then, click **Continue to Review** at the bottom of the page.
+
+<ImageSpotlight
+  alt="On Cart, select Free plan under Choose a plan and Continue to Review at the bottom of the page in the Expo dashboard."
+  src="/static/images/billing/management/billing-10.png"
+  style={{ maxWidth: 640 }}
+/>
+
+- Next, you need to review and confirm that you're switching to the Free plan. Verify that the account you're downgrading is correct (see the _warning_ on this page). After reviewing the details, enable the checkbox and click **Cancel Paid Plan**.
+
+<ImageSpotlight
+  alt="Billing page in Expo dashboard displays the review page for the selected plan to downgrade."
+  src="/static/images/billing/management/billing-11.png"
+  style={{ maxWidth: 640 }}
+/>
+
+- You will be redirected to Stripe's cancel plan screen, where you can confirm your plan. Switching to the Free plan is equivalent to canceling a paid plan at the end of the billing cycle.
```

Comment:
Nit: it should be "where you can cancel your plan". It may be clearer, since confirming a plan might indicate that they're signing up for something.